gtda/images/filtrations.py

```diff
@@ -409,13 +410,16 @@ def __init__(self, n_iterations=None, n_jobs=None):
         self.n_jobs = n_jobs
 
     def _calculate_dilation(self, X):
-        Xd = X * 1.
+        Xd = _dilate(X * 1, 1, self.n_iterations_, 1, self.max_value_)
```

Comment:
`*  1` seems unnecessary here, as we multiply/change type in `_dilate`

Reply:
indeed.